internal/controller/postgrescluster/instance.go

```diff
@@ -1510,7 +1508,7 @@ func (r *Reconciler) reconcileInstanceCertificates(
 			root.Certificate, leafCert.Certificate,
 			leafCert.PrivateKey, instanceCerts)
 	}
-	if err == nil {
+	if err == nil && backupsSpecFound {
```

Comment:
Just double-checking: why don't we need this if using a local volume? Because these certs have to do with talking between repo/instance?